include/internal/time.h

```diff
@@ -67,14 +70,32 @@ static ossl_unused ossl_inline OSSL_TIME ossl_time_infinite(void)
 
 /* Convert time to timeval */
 static ossl_unused ossl_inline
-void ossl_time_time_to_timeval(OSSL_TIME t, struct timeval *out)
+void ossl_time_to_timeval(OSSL_TIME t, struct timeval *out)
 {
 #ifdef _WIN32
     out->tv_sec = (long int)(t.t / OSSL_TIME_SECOND);
 #else
     out->tv_sec = (time_t)(t.t / OSSL_TIME_SECOND);
 #endif
-    out->tv_usec = (t.t % OSSL_TIME_SECOND) / (OSSL_TIME_SECOND / 1000000);
+    out->tv_usec = (t.t % OSSL_TIME_SECOND) / OSSL_TIME_US;
+}
+
+/* Convert timeval to time */
+static ossl_unused ossl_inline
+OSSL_TIME ossl_time_from_timeval(const struct timeval *t)
```

Comment:
It's a little weird that `ossl_time_to_timeval` and `ossl_time_from_timeval` aren't symmetric.  These signatures are perfectly legitimate, for example:

``` C
static ossl_unused ossl_inline struct timeval ossl_time_to_timeval(OSSL_TIME t);
static ossl_unused ossl_inline OSSL_TIME ossl_time_from_timeval(struct timeval t);
```

Reply:
I saw this made an impact.  Cool